.github/workflows/deploy.yml

```
@@ -31,7 +32,7 @@ jobs:
       - name: 압축
         # NODE_MODULES 폴더는 압축하지 않는다.
         # 커밋 해시를 파일명으로 사용하여 고유한 파일명을 생성한다.
-        run: tar -czvf $GITHUB_SHA.tar.gz appspec.yml scripts .next .env package.json package-lock.json
+        run: tar -czvf $GITHUB_SHA.tar.gz appspec.yml scripts .next/standalone .next/static .env public
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC2086:info:1:11: Double quote to prevent globbing and word splitting

(shellcheck)